packages/@o3r/localization/builders/helpers/localization.generator.ts

```diff
@@ -121,6 +135,7 @@ export class LocalizationExtractor {
         }
         return res;
       })
+      .filter(({ref}) => this.options.libraries.every((lib) => !ref.startsWith(lib)))
```

Comment:
would be cleaner to have an optional `option` instead of having defaults values that are not really meaningful.
It would lead to:

```suggestion
      .filter(({ref}) => (this.options.?.libraries || []).every((lib) => !ref.startsWith(lib)))
```